doc/releases/migration-guide-3.7.rst

```diff
@@ -643,6 +647,12 @@ LED Strip
 
 * Made ``update_channels`` function optional and removed unimplemented functions.
 
+MDIO
+====
+
+* :kconfig:option:`CONFIG_MDIO_NXP_ENET_TIMEOUT` is now in units of
```

Comment:
Please add a reference to the github issue that introduced the change ```(:github:`12345`)```

Reply:
hmm, I thought I did this, don't know where it went, I'll try again